### ISSUE TITLE
(AWS) Old subnet id breaks discovery

### ISSUE THREAD

**Issue by patrykorwat** [Cloud-Architects/cloudiscovery#73]
If a resource stores a reference to a removed subnet, the tool stops working. Call to get subnet details should be wrapped with `@exception` annotation.

Such situation happens to SageMaker:
```
Collecting data from Sagemaker Notebook instances...

Error running check SAGEMAKERNOTEBOOK.get_resources. Error message An error occurred (InvalidSubnetID.NotFound) when calling the DescribeSubnets operation: The subnet ID 'subnet-0db79a1feafe30580' does not exist
Collecting data from Security Groups...
```

**Comment by leandrodamascena** [comment]
Discussing this issue in PR #78.

**Comment by leandrodamascena** [comment]
Closed #78
